### PR TITLE
Doc updates to scenario 0 - 2

### PR DESCRIPTION
**Pull request by oybed**
Documentation updates for scenario 0-2 ... will continue reviewing the rest later.

### CODE REVIEW

File: scenario0/README.md [hunk 1]
```diff
@@ -71,7 +77,8 @@ https://master1.example.com:2379 is healthy: successfully committed proposal: to
 
 *_We exectute etcdctl command with prefixed command "docker exec -it etcd_container "etdctl ...". This is because this binary is available ONLY in the running container. It will not work if you try execute same command from host level_*
```
Comment (makentenza) [inline scenario0/README.md]: Not sure why underscores `_` are visible here in the documentation

File: scenario0/README.md [hunk 2]
```diff
@@ -83,66 +90,68 @@ get /openshift.io/templates/openshift/datagrid65-postgresql --prefix"
...
-Now you should see "hell break loose" in the all dashboards.
+At this point, all the dashboards should indicate major failures and clusters in a bad state.
 
 ![alt text](img/img5-granafa-single-survivor.png)
 
 ![alt text](img/img6-hell-got-loose.png)
 
+
+
+####### THIS NEEDS FIXING - WE (Red Hat) DOES NOT SUPPORT SPLITTING A CLUSTER ACROSS MULTIPLE DATA CENTERS #######
```
Comment (makentenza) [inline scenario0/README.md]: As discussed just remove this part as we are not going to open a discussion on this topic during the lab, that's not the goal for this Lab

**Review by makentenza — CHANGES_REQUESTED** [review]
Thanks for the review @oybed, all good from my side. Just a few comments

**Comment by jtudelag** [comment]
LGTM ;)

**Review by jtudelag — APPROVED** [review]
LGTM

File: scenario0/README.md (hunk 1 above)
Comment (oybed) [reply scenario0/README.md]: @makentenza This wasn't something I changed, but I believe the `*_ ....._*` MD will generate bold&italic font. Now that you called out this section, however, I see a typo and will fix it as part of my PR.

File: scenario0/README.md (hunk 2 above)
Comment (oybed) [reply scenario0/README.md]: ok, will do

**Review by makentenza — APPROVED** [review]
LGTM

**Pull request merged**